GeneratorInterface/GenFilters/src/PythiaFilterMultiMother.cc

```diff
@@ -122,3 +123,13 @@ bool PythiaFilterMultiMother::filter(edm::Event& iEvent, const edm::EventSetup&
    return true; } else {return false;}
 
 }
+
+HepMC::FourVector PythiaFilterMultiMother::zboost(const HepMC::FourVector& mom) {
```

Comment:
I'm not sure what's the good way to define the helper function here.
Could you provide some help? Thanks.